upgrade-2.x.sh

```diff
@@ -1026,19 +1026,19 @@ if version_gt "$VERSION_ID" "2.0.6" &&
         tools_path=/tmp/upgrade_tools_extra
         mkdir -p $tools_path
         export PATH=$tools_path:$PATH
-        download_url=https://raw.githubusercontent.com/resin-os/meta-resin/v2.3.0/meta-resin-common/recipes-support/resin-device-progress/resin-device-progress/resin-device-progress
+        download_url=https://raw.githubusercontent.com/balena-os/meta-balena/v2.3.0/meta-balena-common/recipes-support/resin-device-progress/resin-device-progress/resin-device-progress
         CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 -f -s -L -o $tools_path/resin-device-progress $download_url || log WARN "Couldn't download tool from $download_url, progress bar won't work, but not aborting..."
         chmod 755 $tools_path/resin-device-progress
 else
     log "No resin-device-progress fix is required..."
 fi
 
-# Fix for issue: https://github.com/resin-os/meta-resin/pull/864
-# Also includes change from: https://github.com/resin-os/meta-resin/pull/882
+# Fix for issue: https://github.com/balena-os/meta-balena/pull/864
+# Also includes change from: https://github.com/balena-os/meta-balena/pull/882
 if version_gt "$VERSION_ID" "2.0.7" &&
     version_gt "2.7.0" "$VERSION_ID"; then
         log "Fixing supervisor updater..."
-        if CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 --fail --silent -o "/tmp/update-resin-supervisor" https://raw.githubusercontent.com/resin-os/meta-resin/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-resin-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor ; then
+        if CURL_CA_BUNDLE=${TMPCRT} curl --retry 10 --fail --silent -o "/tmp/update-resin-supervisor" https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor ; then
```

Comment:
As before, this gives me a 404; is this expected?

Link: https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor

Try to download with curl:
```
curl -i https://raw.githubusercontent.com/balena-os/meta-balena/40d5a174da6b52d530c978e0cae22aa61f65d203/meta-balena-common/recipes-containers/docker-disk/docker-resin-supervisor-disk/update-resin-supervisor
HTTP/1.1 404 Not Found
Connection: keep-alive
Content-Length: 14
Content-Security-Policy: default-src 'none'; style-src 'unsafe-inline'; sandbox
Strict-Transport-Security: max-age=31536000
X-Content-Type-Options: nosniff
X-Frame-Options: deny
X-XSS-Protection: 1; mode=block
Content-Type: text/plain; charset=utf-8
Via: 1.1 varnish (Varnish/6.0)
X-GitHub-Request-Id: 89EE:2BE1:11934:1712D:5EBEF4A2
Accept-Ranges: bytes
Date: Fri, 15 May 2020 19:59:30 GMT
Via: 1.1 varnish
X-Served-By: cache-sea4476-SEA
X-Cache: HIT, MISS
X-Cache-Hits: 1, 0
X-Timer: S1589572771.712788,VS0,VE7
Vary: Authorization,Accept-Encoding
Access-Control-Allow-Origin: *
X-Fastly-Request-ID: 90ad47b6a4fe18bc28682a0dfcab157f378aa58a
Expires: Fri, 15 May 2020 20:04:30 GMT
Source-Age: 0

404: Not Found
```

Reply:
Good catch! updated.